ui/src/app/components/apps-panel/utils.tsx

```diff
@@ -84,8 +84,12 @@ export const shouldShowApp = (app: UserAppEnvironment): boolean =>
 export const canCreateApp = (app: UserAppEnvironment): boolean =>
   !isVisible(app?.status);
 
+// matches Leonardo code
+// val deletableStatuses: Set[AppStatus] = Set(Unspecified, Running, Error)
```

Comment:
Could you link to the code in the current Leo HEAD commit instead?

Reply:
good idea